src/styles/blocks/catalog.scss

```
@@ -0,0 +1,43 @@
+.catalog {
+  display: grid;
+  justify-items: center;
+  grid-template-columns: repeat(autofill, 200px);
+  gap: 48px;
+
+  max-width: 944px;
+  margin: 0 auto;
+
+  @media (max-width: 488px) {
+    & {
+      grid-template-columns: repeat(1, 1fr);
+    }
+  }
```

If you use media inside a selector - no need to add a redundant clarification
Repeat with other queries as well

```suggestion
  @media (max-width: 488px) {
    grid-template-columns: repeat(1, 1fr);
  }
```